DIDDocument overview

coveloper opened the pull request:
This is just a basic overview.  It needs work still.  

It might make sense to combine the `DID` and `DIDDocument` pages?

robertocarvajal commented on book/section3/dids-and-diddocuments.md:
```diff
@@ -0,0 +1,74 @@
+An Example **DID**:
+`
+did:prism:4a5b5cf0a513e83b598bbea25cd6196746747f065246f1d3743344b4b81b5a74:Cr4BCrsBElsKBmF1dGgwMRJRCglzZWNwMjU2azESBHRlc3QaOmtleTE6Ly8wMjM5MmYxNjc4NmNlNmQ0NzJlOGViNzA4ZWRjMmE3OTFmZGMxNzNkNjVkNTBhODNhMTk3N2I5ZmIwMmU0MjQSWwoGYXV0aDAyElEKCXNlY3AyNTZrMRIEdGVzdBo6a2V5MjovLzAyMzkyZjE2Nzg2Y2U2ZDQ3MmU4ZWI3MDhlZGMyYTc5MWZkYzE3M2Q2NWQ1MGE4M2ExOTc3YjlmYjAyZTQyNA
+`
```
Missing new line in here!

coveloper replied:
added, thanks!

robertocarvajal commented on book/section3/dids-and-diddocuments.md:
```diff
@@ -0,0 +1,74 @@
+    - *Author's note - Specify these in a more concrete way
+- **Services**: An array of `service` descriptions, including URLs that can be used to further interact with the **DID Subject**. These could be API endpoints, for messaging or file storage systems, but any remote service can be added to add value to the **DID**
+- **Controller**: An array of **DIDs** that are allowed to mutate the **DIDDocument**
+- **Proofs**: (*author's note: TBD) Contains cryptographic proofs that assert the validity and integrity of the **DID Document**, ensuring it has not been tampered with.
```
There is no proof section on example resolved DID

robertocarvajal commented on book/section3/dids-and-diddocuments.md:
```diff
@@ -0,0 +1,74 @@
+Let's look at the components of a DID Document:
+
+- **DID**: The **DID** of the **Subject** described by the **DIDDocument**
+- **Context**: Labled as `@context`, this is an array of context specs for the **DIDDocument**.  The first element is usually [https://www.w3.org/ns/did/v1](https://www.w3.org/ns/did/v1) but any other standards used in the DIDDocument should be listed here, for example URIs to definitions for [JSONWebSignature](https://w3id.org/security/suites/jws-2020/v1) or [DIDComm2 Messaging](https://didcomm. ...
```
There is no context tag in the provided example resolved DID

coveloper replied:
Added, thanks!

robertocarvajal commented:
Generally it's good for a draft but just noted some missing content on the example DID payload.. I would say it's fine for a draft because you added some author's notes in there that makes that very clear.

coveloper commented:
> Generally it's good for a draft but just noted some missing content on the example DID payload.. I would say it's fine for a draft because you added some author's notes in there that makes that very clear.

I addressed your feedback, but also revised the text to be more clear.
